examples/IntegrateAndFire_S1967/main.cpp

```diff
@@ -41,6 +41,12 @@ int main(int argc, char **argv) {
   configuration::initialize(argc, argv);
   configuration::observe("v");
 
+   double time_simul=50;
+
+  if(insilico::engine::time_specified){
+    time_simul = insilico::engine::simulation_time;
```

Comment:
Naming, at least in example codes, should be as verbose as possible. In this case `time_simul` can be `simulation_time` or `simulation_duration`. Will reduce the effort to put in comment.